frontend/src/components/DownloadMapData.vue

```diff
@@ -94,6 +100,17 @@ function flattenMapDataJSON(data) {
 }
 
 function downloadMapData() {
+  if (typeof window !== 'undefined' && window.heap) {
+    window.heap.track('Download', {
+      downloadItem: 'Map',
+      selectedSpatialZones: selectedSpatialZones.value.join(', '),
+      selectedTemporalZones: selectedTemporalZones.value.join(', '),
+      selectedProcesses: selectedProcesses.value.join(', '),
+      searchTerm: searchTerm.value
```

Comment:
Good work adding these items in!
One problem I see (that I didn't think of before) is that we are sending lists of node numbers:
https://github.com/user-attachments/assets/e506e04b-7020-42de-b04c-fcbdc19524ee
The problem is that those node numbers WILL ALL CHANGE IN THE FUTURE. (I think?)
If the team adds a taxonomy to the tree, all of the node numbers downstream of that addition will get changed. And that would mean that all of your metrics would be incomparable pre/post making that change.

Better log some permanent identifier for each temporal/spatial zone and each process taxonomy

Reply:
https://github.com/user-attachments/assets/79db2a86-60d2-496f-ad1b-fc3ec5b8590d